app/src/views/subjects/[slug]/lesson/_id.tsx

```diff
@@ -174,7 +182,7 @@ function LessonPage() {
             <div key={test.uuid}>{renderQuestions(test.questions)}</div>
           ))}
           {error && <div className="mb-6 text-red-300">{error}</div>}
-          {tests?.results?.length && (
+          {tests?.results?.length ? (
```

Comment:
I don't think that this change is needed, in this case, they work the same, and imo `&&` is prettier (does not leave a redundant `null` at the end), wdyt? 🎨  😄

Reply:
It was generating that 0 (from length) we want to get rid of 😉

Reply:
Yes, you are right as discussed Slack 👍 
Some information about the test not being present would be perfect 👌